src/utils/timers.ts

```diff
@@ -0,0 +1,3 @@
+import { promisify } from 'node:util';
```

Comment:
Isn't the node: prefix for imports available since a certain v16 minor version?
I'm not sure how can the test pass. When I run it in node 14.16.0 it does give me an error, can we safely move to 'util' in your opinion? That would be awesome!